index.ts

```diff
@@ -75,7 +82,7 @@ function getImages(doc: any, rootUrl: string, imagesPropertyType?: string) {
     } else {
       nodes = doc(`img`);
 
-      if (nodes.length) {
+      if (nodes) {
```

Comment:
is this really correct? since the array can be empty, maybe `if (nodes?.length) {` is a bit more robust?

Reply:
yeah, good point, will fix

Reply:
fixed